src/components/Button/src/Button.vue

```diff
@@ -130,6 +133,14 @@ export default {
 			type: String,
 			default: undefined,
 		},
+		/**
+		 * Tag of button
+		 * @values button, a
+		 */
+		tag: {
+			type: String,
+			default: 'a',
```

Comment:
the default should be `'button'` and we should add a validator that restricts people to passing only `'button'` or `'a'`